Gopkg.toml

```diff
@@ -0,0 +1,54 @@
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
+
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/mitchellh/cli"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/mitchellh/colorstring"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/mitchellh/go-homedir"
+
+[[constraint]]
+  branch = "master"
+  name = "golang.org/x/net"
+
+[[constraint]]
+  branch = "master"
+  name = "golang.org/x/oauth2"
+
+[[constraint]]
+  name = "gopkg.in/yaml.v2"
+  version = "2.2.1"
```

Comment:
Is there a reason this version needs to be pinned? Seems like `HEAD` for the dependencies is OK for now, especially since this is very early alpha and likely to change. If/when we get to a point where we proliferate a bunch of dependencies and require specific versions of pinning we can add it, but I think this is unnecessary overhead for now.

Reply:
no specific reason this version needs to be pinned at all. completely agree where `HEAD` is fine for a smaller, alpha codebase.